main.css

```diff
@@ -0,0 +1,769 @@
+  line-height: 25px;
+  color: #f42;
+}
+
+footer {
+  width: 100%;
+}
+
+.first-footer {
+  background-color: #333;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr;
+  justify-items: center;
+  padding: 3em 0;
+}
+
+.social-icons {
+  display: flex;
+  list-style: none;
+  padding: 1em 0 0 auto;
+}
+
+.social-icons li {
+  padding: 0 0.5em;
+}
+
+.social-list {
+  display: flex;
+  list-style: none;
+  padding: 1em 0;
+}
+
+.second-footer {
+  background-color: black;
+  text-align: center;
+  padding: 2em 0;
+}
+
+.second-footer p {
+  color: #999;
+  font-size: 0.9em;
+}
+
+.second-footer span {
+  color: red;
+  font-size: 1.5em;
+}
+
+.bar-one a:hover {
+  color: white;
+}
+
+.bar-two a:hover,
+.funds-address a:hover,
+.grid-address a:hover,
+.news p a:hover,
+.list-grid h2 a:hover {
+  color: black;
+}
+
+.list-2 button:hover {
+  background-color: #609;
+  cursor: pointer;
+}
+
+@media only screen and (max-width: 1024px) and (min-width: 601px) {
+  .removable-img {
+    display: block;
+  }
+
+  .bar-two {
+    justify-content: space-between;
+  }
+
+  .bar-two .list-2 {
+    display: none;
+  }
+
+  .bar-one {
+    display: none;
+  }
+
+  .removable-icons {
+    display: block;
+    padding: 1.2em 0.5em;
+  }
+
+  .removable-icons a {
+    padding: 0 1em;
+    font-size: 1em;
+  }
+
+  .bar-three a {
+    font-size: 0.8em;
+  }
+
+  main {
+    display: grid;
+    grid-template-columns: 4fr repeat(2, 4fr);
+    margin-top: 8em;
+  }
+
+  .sec-1 {
+    grid-column: 1/ 3;
+    grid-row: 1/ 3;
+  }
+
+  .sec-head img {
+    display: none;
+  }
+
+  .sec-foot h1 {
+    font-size: 2em;
+  }
+
+  .sec-2 {
+    grid-column: 3;
+    grid-row: 1;
+  }
+
+  .sec-2 h1 {
+    margin-top: 8em;
+  }
+
+  .sec-3 {
+    grid-column: 3;
+    grid-row: 2;
+  }
+
+  .sec-2 h1,
+  .sec-3 h1 {
+    font-size: 1em;
+  }
+
+  .gridded {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-areas: "head head head";
+    margin: 1em 0;
+    padding: 0 1em;
+  }
+
+  .grid-address {
+    grid-area: head;
+  }
+
+  .fund-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-areas: "head head head";
+    margin: 1em 0;
+    padding: 0 1em;
+  }
+
+  .funds-address {
+    grid-area: head;
+    display: block;
+    font-size: 0.9em;
+  }
+
+  .removable {
+    display: none;
+  }
+
+  .list-grid {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    margin: 5em 0;
+    padding: 0 1em;
+  }
+
+  .social-icons li {
+    padding: 0 0.2em;
+  }
+
+  .social-icons a {
+    padding: 8px 10px;
+  }
+
+  .social-list a {
+    padding: 0 0.2em;
+  }
+}
+
+@media only screen and (min-width: 320px) and (max-width: 600px) {
+  .removable-img {
+    display: block;
+  }
+
+  .bar-two {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .bar-one {
+    display: none;
+  }
+
+  .bar-two .list-1,
+  .bar-two .list-2 {
+    display: none;
+  }
```

Comment:
As requested by the previous TSE, please use the DRY principle to optimize your CSS file